sdk/core/Azure.Core/perf/DownloadHttpClientTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class DownloadHttpClientTest : PerfTest<DownloadTestOptions>
+    {
+        private readonly HttpClient _client;
```

Comment:
Do you want to create one client and server per instance of the test class, or create a single static instance and share?  This is important when using parallel > 1.

Reply:
In other tests we create SDK clients per instance.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Files.DataLake/perf/Azure.Storage.Files.DataLake.Perf/Scenarios/Upload.cs#L36 in particular.

Reply:
For the client, our test should do whatever we would recommend to customers wanting the best performance.  In a real-world customer web app, would we recommend sharing a single client (like is recommended with `System.Net.HttpClient`), or just creating a new client whenever you need it.

For the server, I think you only want one regardless of how many test instances are running in parallel, right?